tests/framework/config/client.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import clientv3 "go.etcd.io/etcd/client/v3"
+
+type GetConfig struct {
```

Comment:
I would keep the name "Options" / "GetOptions". That's the abstraction used in client... 
and will be more descriptive ... even if used in context of etcdctl.

Reply:
(just naming change, not representation)

Reply:
Done